packages/bpk-component-select/src/BpkSelect.module.scss

```diff
@@ -42,38 +44,42 @@
   }
 
   &--with-image {
-    padding-left: $bpk-spacing-xl;
+    padding-left: bpk-spacing-xxl();
 
     @include bpk-rtl {
-      padding-right: $bpk-spacing-xl;
-      padding-left: $bpk-select-padding-right;
+      padding-right: bpk-spacing-xxl();
+      padding-left: bpk-spacing-xxl();
 
       @media screen\0 {
-        padding-right: $bpk-spacing-xl !important; /* stylelint-disable-line declaration-no-important */
-        padding-left: $bpk-spacing-sm !important; /* stylelint-disable-line declaration-no-important */
+        /* stylelint-disable-next-line declaration-no-important */
+        padding-right: bpk-spacing-xxl() !important;
+        /* stylelint-disable-next-line declaration-no-important */
+        padding-left: bpk-spacing-base() !important;
       }
     }
 
     @media screen\0 {
-      padding-left: $bpk-spacing-xl !important; /* stylelint-disable-line declaration-no-important */
+      /* stylelint-disable-next-line declaration-no-important */
```

Comment:
I appreciate you didn't add these, but every lint-disable comment should be accompanied by an explanation.
I assume the reason for these is that we need to ensure we override the browser styles?

Reply:
A single explanation at the top of the file should be sufficient

Reply:
Good shout! Better to improve things now as we are making changes. Yeah I assume that is why, its an IE override thing but will put that in a comment at the top!